TechLead/src/main/java/first/Client.java

```diff
@@ -0,0 +1,88 @@
+package first;
+
+
+import java.io.*;
+import java.net.InetAddress;
+import java.net.Socket;
+
+public class Client {
+
+    private static int serverPort; // здесь обязательно нужно указать порт к которому привязывается сервер.
+    private static String address = "127.0.0.1"; // это IP-адрес компьютера, где исполняется наша серверная программа.
+
+    private static Socket socket; // Здесь указан адрес того самого компьютера где будет исполняться и клиент.
+
+    public static int i;
+    private static InputStream socketInputStream;
+    private static OutputStream socketOutputStream;
+
+
+
+    public static void startClient() throws IOException {
+
+        try {
+            i=5;
+            //serverPort = Integer.parseInt(Utils.getProperty("port"));
+            serverPort = 8081;
+            InetAddress ipAddress = InetAddress.getByName(address); // создаем объект который отображает вышеописанный IP-адрес.
+
+            socket = new Socket(ipAddress, serverPort); // создаем сокет используя IP-адрес и порт сервера.
+            System.out.println("add socket: " + address + ":" + serverPort );
+
+            // Берем входной и выходной потоки сокета, теперь можем получать и отсылать данные клиентом.
+            socketInputStream = socket.getInputStream();
+            socketOutputStream = socket.getOutputStream();
+
+            // Конвертируем
+            // потоки в другой тип, чтоб легче обрабатывать текстовые сообщения.
+            DataInputStream dataInputStream = new DataInputStream(socketInputStream);
```

Comment:
DataInputStream dataInputStream = new DataInputStream(socket.getInputStream()); is simpler